### DISCUSSION TITLE
Possible to have a statblock in a table?

### DISCUSSION THREAD

**Discussion by InStim** [Obsidian-TTRPG-Community/fantasy-statblocks#113] [Q&A]
Is it possible to have a statblock in a table cell? I am using Dice Roller to do wandering monsters with a table roller, and I was hoping to have it return the statblock of monsters saved in the bestiary.

I am new to Obsidian and it's markdown, so I was wondering if I was missing something,

Thanks for any insight!

**Comment by sigrunixia** [comment]
In a standardized table cell, no.

For more advanced options, there is the usage of dataviewjs to read the api of both stat blocks and dice roller. 

Note to self: Dataview Integration Documentation
